operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/AbstractSystemConfigurationImpl.java

```diff
@@ -152,7 +152,7 @@ public long getCoreCosMaxResources() {
 
     private static final long defaultCoreCosMaxResources() {
         final String PATH = "fhirServer/bulkdata/core/cos/maxResourcesThreshold";
-        return 1000 * 1000 * FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COSFILE_MAX_RESOURCESNUMBER);
+        return 1000 * FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COSFILE_MAX_RESOURCESNUMBER);
```

Comment:
do we need to use `1000L` to force it to autobox to a long?  or just to make it more clear?

Reply:
sure